eng/pipelines/runtime.yml

```diff
@@ -695,6 +697,7 @@ jobs:
       condition: >-
         or(
           eq(dependencies.checkout.outputs['SetPathVars_mono.containsChange'], true),
+          eq(dependencies.checkout.outputs['SetPathVars_runtimetests.containsChange'], true),
```

Comment:
You will also need to make sure mono is built (at least for OSX), same for CoreCLR Release, Libraries (Which I think already does).

Reply:
I added it for mono, it looks like CoreCLR and Libraries are always built, right? 
https://github.com/dotnet/runtime/blob/0fa0b905f65e57f5eb64dd556a07ead522930c8a/eng/pipelines/runtime.yml#L183
https://github.com/dotnet/runtime/blob/0fa0b905f65e57f5eb64dd556a07ead522930c8a/eng/pipelines/runtime.yml#L357

Reply:
Yeah, those are always built.